### ISSUE TITLE
Zulip doesn't store conversations? Broken link in Tour of Selected equations

### ISSUE THREAD

**Issue by NooneAtAll3** [teorth/equational_theories#549]
I've been browsing [Tour of Selected Equations](https://github.com/teorth/equational_theories/wiki/Tour-of-selected-equations), and the "Equations 65, 1491: "Asterix and Obelix"" paragraph in it has a following quote
> see [this ongoing discussion](https://leanprover.zulipchat.com/#narrow/stream/458659-Equational/topic/Equation.2065.20-.3E.20Equation.20359)"

link is dead because... Zulip doesn't seem to store old conversations?

(I'm not part of the project, but I've been participating in BBchallenge, so this kind of non-archiveability feels appalling)

But at the very least such dead links shouldn't stay public - so I suggest global search and prune of Zulip links in the repo (other than in Readme for the whole server, I suppose)

**Comment by vlad902** [comment]
What I'm guessing happened here, is that topic was renamed and that the link therefore changed to [this](https://leanprover.zulipchat.com/#narrow/stream/458659-Equational/topic/Equation.2065.20-.3E.20left.20cancellability). I am not particularly familiar with Zulip, but googling around it looks like this is a known issue with Zulip being tracked [here](https://github.com/zulip/zulip/issues/21505) and that support for permalinks is coming shortly in the future, but this unfortunate behavior is the current state of things.

**Comment by vlad902** [comment]
So reading that Zulip issue further, only links to topics are not currently permalinks, links to specific messages are. Links to topics end in /topic/NAME and links to messages end in /topic/NAME/near/NUMBER. I'll go ahead and open a PR shortly to update links to topics in the repo. I think only @teorth can update the wiki links.

**Comment by teorth** [comment]
I've fixed that specific wiki link, thanks for pointing it out.

**Comment by Chessing234** [comment]
PR #1444 adds the missing /near/ anchor to the Database of implications links in README and home_page/index.md — they were left as topic-only URLs after #554.

**Comment by Chessing234** [comment]
Also opened #1445 for the remaining topic-only link in commentary/Equation879.md.

**Comment by Chessing234** [comment]
Opened #1446 to add the missing permalink on the Equation677 finite-implication link.

**Comment by Chessing234** [comment]
Opened #1447 for the Dupont/Dupond discussion link in Equation63 commentary.

**Comment by Chessing234** [comment]
PR opened for the matching topic-only Dupont/Dupond link in `commentary/Equation1692.md` — #1447 fixed the same URL in `Equation63.md` but this duplicate was still missing the `/near/477121484` anchor.
